DistFiles/localizations/Chorus.en.tmx

```diff
@@ -1,922 +1,945 @@
-		<seg>Show characters</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="ServerSettingsControl.URL">
-	  <tuv xml:lang="en">
-		<seg>URL</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="ServerSettingsDialog.WindowTitle">
-	  <tuv xml:lang="en">
-		<seg>Send/Receive Server Settings...</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncControl.ChooseRepos">
-	  <tuv xml:lang="en">
-		<seg>Choose Respositories</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncControl.Log">
-	  <tuv xml:lang="en">
-		<seg>Log</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncControl.LogBox">
-	  <tuv xml:lang="en">
-		<seg>Box:Verbose:</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncControl.PeopleToSendReceive">
-	  <tuv xml:lang="en">
-		<seg>Attempt to Send/Receive with these people, devices, and servers:</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncControl.SendReceiveButton">
-	  <tuv xml:lang="en">
-		<seg>Send/Receive</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncControl.Tasks">
-	  <tuv xml:lang="en">
-		<seg>Tasks</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncDialog.WindowTitle">
-	  <tuv xml:lang="en">
-		<seg>Send/Receive</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncPanel.SendReceiveProjectChanges">
-	  <tuv xml:lang="en">
-		<seg>Send/Receive Project Changes</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncStartControl.Checking">
-	  <tuv xml:lang="en">
-		<seg>Checking...</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncStartControl.CommitMessage">
-	  <tuv xml:lang="en">
-		<seg>Label this point in the project history (Optional) :</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncStartControl.Diagnostics">
-	  <tuv xml:lang="en">
-		<seg>Diagnostics</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncStartControl.UseInternetButton">
-	  <tuv xml:lang="en">
-		<seg>&amp;Internet</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncStartControl.UseLocalNetworkButton">
-	  <tuv xml:lang="en">
-		<seg>&amp;Chorus Hub</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="SyncStartControl.UseUSBButton">
-	  <tuv xml:lang="en">
-		<seg>&amp;USB Flash Drive</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="TargetFolderControl.Download">
-	  <tuv xml:lang="en">
-		<seg>&amp;Download</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="TargetFolderControl.LocalFolder">
-	  <tuv xml:lang="en">
-		<seg>Name for the folder on your computer</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="TroubleshootingView.CopyToClipboard">
-	  <tuv xml:lang="en">
-		<seg>copy to clipboard</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="TroubleshootingView.RunDiagnosticsButton">
-	  <tuv xml:lang="en">
-		<seg>Run Diagnostics</seg>
-	  </tuv>
-	</tu>
-	<tu tuid="TroubleshootingView.TroubleshootingView._statusLabel">
-	  <tuv xml:lang="en">
-		<seg>label1</seg>
-	  </tuv>
-	</tu>
+    <tu tuid="AnnotationEditorView.Add">
+      <tuv xml:lang="en">
+        <seg>&amp;Add &amp;&amp; &amp;OK</seg>
+      </tuv>
+    </tu>
+    <tu tuid="AnnotationEditorView.AddAndOK">
+      <prop type="x-nolongerused">true</prop>
+      <tuv xml:lang="en">
+        <seg>&amp;Add &amp;&amp; &amp;OK</seg>
+      </tuv>
+    </tu>
+    <tu tuid="AnnotationEditorView.AddAndOk">
```

Comment:
I removed this line and it was re-added as is by the Update code. There must be a button in the actual dialog code named AddAndOK with Cancel as the text. I'm not going to clean up chorus dialogs as part of this change. Good catch though.